### ISSUE TITLE
Test kısmında verilen hata

### ISSUE THREAD

**Issue by eyupkagansahin** [cemkod/e-bliss-rapgen#3]
9. kod kümesini çalıştırdığım esnada aşağıdaki hatayı alıyorum ne yapmak gerekiyor?

from transformers import AutoTokenizer, AutoModelWithLMHead
import torch

tokenizer = AutoTokenizer.from_pretrained("gorkemgoknar/gpt2-small-turkish")
model = AutoModelWithLMHead.from_pretrained("drive/MyDrive/Rapgen/gpt2-trap-1epoch")
 GPT-2 en fazla 1024 tokenlik bir dizi uretebiliyor
tokenizer.model_max_length=1024 

**/usr/local/lib/python3.6/dist-packages/transformers/models/auto/modeling_auto.py:925: FutureWarning: The class `AutoModelWithLMHead` is deprecated and will be removed in a future version. Please use `AutoModelForCausalLM` for causal language models, `AutoModelForMaskedLM` for masked language models and `AutoModelForSeq2SeqLM` for encoder-decoder models.
  FutureWarning,**

**Comment by cemkod** [comment]
O bir hata degil, sadece uyari. kullanilan bir class'in gelecekte kaldiralacigini soyluyor. Calismasina etki etmemesi gerekli.
